lib/circlex/emulator/state/bank_account_state.ex

```diff
@@ -48,7 +48,8 @@ defmodule Circlex.Emulator.State.BankAccountState do
        billing_details: billing_details,
        bank_address: bank_address,
        create_date: State.next(:date),
-       update_date: State.next(:date)
+       update_date: State.next(:date),
+       virtual_account_number: State.next(:virtual_account_number)
```

Comment:
Do we _always_ generate a virtual account number or is this some kind of opt-in thing?